### PR TITLE
Add 'info' command to 'mfg' submenu.

### PR DESCRIPTION
**Pull request by xyan264**

### CODE REVIEW

File: cli/mfg.c [hunk 1]
```diff
@@ -84,8 +85,168 @@ static int ping(int argc, char **argv)
...
+
+	argconfig_parse(argc, argv, desc, opts, &cfg, sizeof(cfg));
+
+	ret = switchtec_get_device_info(cfg.dev, &phase_id, NULL, NULL);
+	if (ret < 0) {
+		switchtec_perror("mfg info");
+		return ret;
+	}
+
+	printf("Current Boot Phase: \t\t\t");
```
Comment (lsgunth) [inline cli/mfg.c]: Looks like this could be made common with the code in ping()

Comment (xyan264) [reply cli/mfg.c]: Good idea. Moved duplicate switch...case to phase_id_to_string().

File: lib/mfg.c [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+ * @param[out] info	Serial number and security version info
+ * @return 0 on success, error code on failure
+ */
+int switchtec_sn_ver_get(struct switchtec_dev *dev,
+			 struct switchtec_sn_ver_info *info)
+{
+	int ret;
+
+	ret = switchtec_cmd(dev, MRPC_SN_VER_GET, NULL, 0,
+			info,
```
Comment (kelvin-cao) [inline lib/mfg.c]: indent

File: lib/mfg.c [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+	state->i2c_port = (reply.cfg >> SWITCHTEC_I2C_PORT_BITSHIFT) & 0xf;
+	state->i2c_addr = (reply.cfg >> SWITCHTEC_I2C_ADDR_BITSHIFT) & 0x7f;
+	state->i2c_cmd_map =
+		(reply.cfg >> SWITCHTEC_CMD_MAP_BITSHIFT) & 0xfff;
+
+	state->public_key_exponent = reply.public_key_exponent;
+	state->public_key_num = reply.public_key_num;
+	state->public_key_ver = reply.public_key_ver;
+	memcpy(state->public_key, reply.public_key,
+		SWITCHTEC_KMSK_NUM * SWITCHTEC_KMSK_LEN);
```
Comment (kelvin-cao) [inline lib/mfg.c]: indent

File: cli/mfg.c [hunk 4]
```diff
@@ -65,27 +80,159 @@ static int ping(int argc, char **argv)
...
+
+	argconfig_parse(argc, argv, desc, opts, &cfg, sizeof(cfg));
+
+	ret = switchtec_get_device_info(cfg.dev, &phase_id, NULL, NULL);
+	if (ret < 0) {
+		switchtec_perror("mfg info");
+		return ret;
+	}
+	printf("Current Boot Phase: \t\t\t%s\n",
+			phase_id_to_string(phase_id));
```
Comment (kelvin-cao) [inline cli/mfg.c]: This can be in the same line of printf

**Pull request merged**